IPython/core/tests/test_ultratb.py

```diff
@@ -442,3 +442,48 @@ def eggs(f, g, z=globals()):
     except:
         handler(*sys.exc_info())
     buff.write('')
+
+
+# ----------------------------------------------------------------------------
+
+# module Test deprecated '_tb_highlight' & '_tb_highlight_style'
+
+
+def test_getter_deprecation_warning_tb_highlight(recwarn):
+    VerboseObj = VerboseTB()
+    value = VerboseObj._tb_highlight
+    w = recwarn.pop(DeprecationWarning)
```

Comment:
Just in case, another syntax for this: https://docs.pytest.org/en/latest/reference/reference.html#pytest-warns